web/src/components/storage/ProposalResultTable.jsx

```diff
@@ -89,21 +90,21 @@ const DeviceCustomSize = ({ item, devicesManager }) => {
   const sizeBefore = isResized ? devicesManager.systemDevice(device.sid).size : item.size;
 
   return (
-    <div className="split">
+    <Flex direction={{ default: "row" }} gap={{ default: "gapXs" }}>
+      <DeviceSize item={item} />
       <If
         condition={isResized}
         then={
-          <Tag variant="orange">
```

Comment:
Thus, <Tag> component is now only used by storage/device-utils.jsx. Something to check and drop

Reply:
Dropped.